docs/reference/components/sources/postgresql_metrics.cue

```diff
@@ -0,0 +1,391 @@
+package metadata
+
+components: sources: postgresql_metrics: {
+	title:       "PostgreSQL Metrics"
+	description: "[PostgreSQL][urls.postgresql] PostgreSQL is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance."
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+	}
+
+	features: {
+		collect: {
+			checkpoint: enabled: false
+			from: {
+				service: {
+					name:     "PostgreSQL Server"
+					thing:    "an \(name)"
+					url:      urls.postgresql
+					versions: null
+				}
+
+				interface: {
+					socket: {
+						direction: "outgoing"
+						protocols: ["tcp", "unix"]
+						ssl: "optional"
+					}
+				}
+			}
+		}
+		multiline: enabled: false
+	}
+
+	support: {
+		targets: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: []
+
+		warnings: []
+		notices: []
+	}
+
+	installation: {
+		platform_name: null
+	}
+
+	configuration: {
+		endpoints: {
+			description: "PostgreSQL server endpoint in libpq-style connection strings."
+			required:    true
+			type: array: {
+				items: type: string: examples: ["postgresql://postgres:vector@localhost:5432/postgres"]
+			}
+		}
+		scrape_interval_secs: {
+			description: "The interval between scrapes."
+			common:      true
+			required:    false
+			type: uint: {
+				default: 15
+				unit:    "seconds"
+			}
+		}
+		namespace: {
+			description: "The namespace of metrics. Disabled if empty."
+			common:      false
+			required:    false
+			type: string: default: "postgresql"
+		}
+		include_databases: {
+			description: """
+				A list of databases to match against `datname` column for which you want to collect metrics from.
+				This can beused in conjunction with [`exclude_databases`](#exclude_databases).
+				In case if `exclude_databases` is not empty databases from there will be removed from `include_databases`.
```

Comment:
```suggestion
				A list of databases to match against the `datname` column for which you want to collect metrics from. If not set, metrics will be collected from all databases. This can be used in conjunction with [`exclude_databases`](#exclude_databases).
```